core-web/apps/dotcms-ui/.storybook/main.js

```diff
@@ -11,6 +11,9 @@ module.exports = {
         '../src/**/*.stories.@(js|jsx|ts|tsx)',
     ],
     addons: [...rootMain.addons],
+    features: {
+        previewMdx2: true,
+    },
```

Comment:
What is this for?

Reply:
Is to force use "@mdx-js/react": "^2.1.2 in storybook